src/interactions/panZoomInteraction.ts

```diff
@@ -11,16 +11,14 @@ namespace Plottable.Interactions {
     return center - (center - value) * zoom;
   }
 
-  export type TransformableScale = Plottable.Scale<any, number> & Plottable.Scales.TransformableScale;
-
   export class PanZoom extends Interaction {
     /**
      * The number of pixels occupied in a line.
      */
     private static _PIXELS_PER_LINE = 120;
 
-    private _xScales: Utils.Set<TransformableScale>;
-    private _yScales: Utils.Set<TransformableScale>;
+    private _xScales: Utils.Set<TransformableScale<any, number>>;
```

Comment:
nit: is `any` the right type here? Would it be valuable to give PanZoom a type parameter for this instead?